package.json

```diff
@@ -23,14 +24,20 @@
   },
   "homepage": "https://github.com/emp-daisy/HelloBooksLib#readme",
   "dependencies": {
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "jest": "^24.8.0",
+    "supertest": "^4.0.2"
```

Comment:
I see supertest

Reply:
Supertest is just like chai
I left it there in case someone decides to write a special test with supertest
Like testing for an Invalid json object for instance

Reply:
Yeah. I know. But I just felt we should only install what we need. As in when we need it, we install it.

Reply:
You've got a point